### ISSUE TITLE
Videos appear to import, but user can't find them

### ISSUE THREAD

**Issue by lucastbelem** [themeetinghouse/web#1239]
Here is what happens:
- User runs the manual importer
- Video appears and user enters metadata
- User clicks "save"
- User sees "saving". It never gets to "saved"
- User goes to the website to see if the video is where it's supposed to be (either teaching video in the main baby hero or livestream video in the sunday livestreams carousel at the bottom of the teaching page)
- User does not see the video in the expected spot
- User returns to the importer and attempts to import the video again
- Media Manager reports to the user that the video already exists
- User cannot find the video in the Media Manager

Known videos for which this issue seems applicable:
Livestream Episode 4 of "Being. Whole."

**Comment by itbel** [comment]
This appears to be working.... the table UI isnt the most clear but the video in question imported without issue for me.

**Comment by lucastbelem** [comment]
@itbel did you fix the refresh requirement to see the video? Here is what Aaron reported last night:

• When I clicked [Import], filled all the relevant fields, and pressed save; I saw this error message (attached) in the Console.
• However, after refreshing the page, the video appeared among "Unlisted", minus the details I'd added. When I filled in the fields and pressed save, I saw no error message, and I confirmed that the video appeared on the /teaching page.
• For the rest of the ACTS videos, I proceeded to refresh the page after importing, without filling anything in. Then, after reselecting it under "Unlisted", I filled in the fields, and saved; everything worked fine.
